centreon/www/front_src/src/AuthenticationTokens/atoms.ts

```diff
@@ -15,3 +15,5 @@ export const filtersAtom = atomWithStorage<Filter>(
 
 export const modalStateAtom = atom<ModalState>({ isOpen: false, mode: 'add' });
 export const tokenAtom = atom<string | null>(null);
+
+export const isWelcomePageDisplayedAtom = atom<boolean>(true);
```

Comment:
```suggestion
export const isWelcomePageDisplayedAtom = atom(true);
```
The type is inferred